project/settings.py

```diff
@@ -14,11 +14,10 @@
 
 from django.contrib.messages import constants as messages
 
-MODE = "prod"  # Overridden by local settings.
 ENVIRONMENT = os.environ.get("ENVIRONMENT", "develop")
 
 # Build paths inside the project like this: os.path.join(BASE_DIR, ...)
-BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+BASE_DIR = os.path.dirname(os.path.abspath(__file__))
 # root path for ember builds
 EMBER_BUILD_ROOT_PATH = os.path.join(BASE_DIR, "../ember_build")
```

Comment:
Let's just completely lean into [using pathlib](https://adamj.eu/tech/2020/03/16/use-pathlib-in-your-django-project/)

Reply:
See Above